pages/common/acme.sh.md

```diff
@@ -0,0 +1,36 @@
+# acme.sh
+
+> Shell script implementing ACME client protocol, an alternative to certbot.
+> More information: <https://github.com/acmesh-official/acme.sh>.
+
+- Issue a certificate using webroot mode:
+
+`acme.sh --issue --domain {{example.com}} --webroot {{/path/to/webroot}}`
+
+- Issue a certificate using standalone mode using port 80:
+
+`acme.sh --issue --standalone --domain {{example.com}}`
+
+- Issue a certificate using standalone TLS mode using port 443:
+
+`acme.sh --issue --alpn --domain {{example.com}}`
+
+- Issue a certificate using a working Nginx configuration:
+
+`acme.sh --issue --nginx --domain {{example.com}}`
+
+- Issue a certificate using a working Apache configuration:
+
+`acme.sh --issue --apache --domain {{example.com}}`
+
+- Issue a wildcard (\*) certificate using a manual DNS mode:
+
+`acme.sh --issue --dns --domain {{example.com}}`
+
+- Issue a certificate using an automatic DNS API mode:
+
+`acme.sh --issue --dns {{dns_api}} --domain {{example.com}}`
+
+- Install certificate file into location specified by its parameter (not displayed):
+
+`acme.sh --install-cert --domain {{example.com}} --reloadcmd {{reload_commad}}`
```

Comment:
I have shortened it especially because it could become too long, as we are still discussing above.
As far as I can tell right now, we have at least three options:
1. drop the example as it is too long (I am strongly opposed, but might be sufficient)
2. include relatively short example with generic values (would serve more as a reminder)
3. include full length example (most valuable, might break some clients without multiline code support)